### ISSUE TITLE
Tab navigation not working properly

### ISSUE THREAD

**Issue by drumilpatel2000** [Submitty/Submitty#3539]
**Describe the bug**
Some of the focus able element cannot be accessed using tabs

**Expected behavior**
Each focus able element should be accessed using tab index 

**To Reproduce**
1. Open discussion forum in Instructor
2. Using tab navigate to `Create Thread`  
3. You can easily see the error. It jumps from `Create Thread` to `Stats` 
 

**Configuration**
All 

**Screenshots**
![ezgif com-video-to-gif(15)](https://user-images.githubusercontent.com/42701709/55544673-3189e480-56e9-11e9-90e8-8f98b93b8cf2.gif)

**Additional context**

**Comment by bmcutler** [comment]
@xprilion 
Tab navigation is much improved in #3859, but there are still some issues.

* initially nothing should have focus (this is how the other pages work). Then when tab is pressed for the first time focus should highlight the 'submitty' in the breadcrumb.  Currently focus starts on Submitty in the breadcrumb (sorry I think I mis-spoke earlier).  

* The ordering of "More" and "Create thread" is swapped.  It should go left to right across the bar.

* I think if we press 'enter' when focus is on a specific thread, then when that thread loads focus should be on the posts in the thread.  Currently it jumps back to the breadcrumb and we have a long way to navigate.  Maybe this will be addressed when we have dynamic thread loading?  We can wait to fix the tabs until that implementation is done.

* Once focus enters the posts within a thread, it is currently skipping many of the buttons.  It should visit every reply, show history, show full user information, edit, delete, merge (as appropraite, if instructor), hide replies, link code, attachment, upload attachment, anonymous, reply to all.  Etc.  I may have missed some.  In general they should go from left to right top to bottom visually.

**Comment by xprilion** [comment]
Getting onto this now.

**Comment by elihschiff** [comment]
I think this issue should be closed now that #4899 has been merged. #5039 is still open making this issue redundant. The specifics problem from this issue has been addressed by #4899.

If somebody has a reason to keep this issue open, please let me know, otherwise I will close this issue.
